config/dev.exs

```diff
@@ -39,5 +39,6 @@ config :pairmotron, Pairmotron.Repo,
   username: System.get_env("PG_USER"),
   password: System.get_env("PG_PASSWORD"),
   database: "pairmotron_dev",
-  hostname: "localhost",
+  hostname: System.get_env("PG_HOST"),
```

Comment:
I suggest `System.get_env("PG_HOST") || "localhost"` for this file and test.exs. That allows for overriding but prevents existing developers from needing another env var.